synapse/storage/databases/main/registration.py

```diff
@@ -70,7 +70,12 @@ def _default_token_owner(self):
 
 
 class RegistrationWorkerStore(CacheInvalidationWorkerStore):
-    def __init__(self, database: DatabasePool, db_conn: Connection, hs: "HomeServer"):
+    def __init__(
+        self,
+        database: DatabasePool,
+        db_conn: LoggingDatabaseConnection,
```

Comment:
I remember you and @richvdh having a conversation about switching these from `Connection` to `LoggingDatabaseConnection`, but do not remember the OK. Are we OK making this change?

Reply:
Yeah, I think we are

Reply:
he convinced me I think